datasets/open_targets/pipelines/_images/copy_bq_datasets/script.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import operator
+import os
+import time
+import typing
+
+from google.api_core.exceptions import ResourceExhausted
+from google.cloud import bigquery_datatransfer_v1
+from google.protobuf.timestamp_pb2 import Timestamp
+
+RETRY_DELAY = 10
+
+
+class TimeoutError(Exception):
+    """Raised when the BQ transfer jobs haven't all finished within the allotted time"""
+
+    pass
+
+
+def main(
+    source_project_id: str,
+    target_project_id: str,
+    service_account: str,
+    dataset_name: str,
+    dataset_versions: typing.List[str],
```

Comment:
We won't need `dataset_versions` here because we only have the `platform` dataset to copy. 

For an example of how you can replace this with, say `source_dataset` and `target_dataset`, see https://github.com/GoogleCloudPlatform/public-datasets-pipelines/blob/main/datasets/google_trends/pipelines/_images/bq_data_transfer/script.py 

I'm using `expected_tables` there but you can easily modify that to use source and target dataset parameters.

Reply:
@adlersantos done